### DISCUSSION TITLE
Migrate your Shopify store assets to a new store using File API | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#69] [Announcements]
# Migrate your Shopify store assets to a new store using File API | Leo's dev blog – stories, insights, and ideas

Learn how to migrate your assets from an old Shopify store to a new one in a few simple steps. This guide will show you how to retrieve and download your images using GraphQL and Node.js, and upload them to your new store with ease.

https://www.leohuynh.dev/blog/migrate-shopify-store-files-to-new-store-using-file-api

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺

**Comment by soorajkabeer** [comment]
I haven't had the need to migrate assets yet, but I've often wondered about the process if it ever comes up. This guide is clear, and I can see how helpful it will be for future projects. Great to know there's a solid approach. Thanks for sharing!

**Reply by hta218** [reply]
Great it could help others, thanks so much @soorajkabeer!
